monkey/agent_plugins/exploiters/smb/src/smb_exploit_client.py

```diff
@@ -1,264 +1,177 @@
 import logging
-import ntpath
-from io import BytesIO
 from pathlib import PurePath
-from typing import Any, Dict, Optional, Sequence, Tuple
-
-from impacket.dcerpc.v5 import scmr, srvs, transport
-from impacket.smbconnection import SMBConnection
+from typing import List, Set, Tuple, Type
 
 from common import OperatingSystem
 from common.credentials import Credentials
-from common.event_queue import IAgentEventPublisher
-from common.types import Event
-from infection_monkey.exploit.tools import RemoteFileCopyError
-from infection_monkey.exploit.tools.helpers import get_agent_dst_path
+from common.tags import (
+    T1021_ATTACK_TECHNIQUE_TAG,
+    T1105_ATTACK_TECHNIQUE_TAG,
+    T1110_ATTACK_TECHNIQUE_TAG,
+    T1135_ATTACK_TECHNIQUE_TAG,
+    T1210_ATTACK_TECHNIQUE_TAG,
+    T1569_ATTACK_TECHNIQUE_TAG,
+)
+from common.types import NetworkPort
+from infection_monkey.exploit.tools import (
+    IRemoteAccessClient,
+    RemoteAuthenticationError,
+    RemoteCommandExecutionError,
+    RemoteFileCopyError,
+)
 from infection_monkey.i_puppet import TargetHost
 
-from .smb_command_builder import build_smb_command
+from .smb_client import ShareInfo, SMBClient
```

Comment:
why is plugin source directory not a python package?

Reply:
It doesn't need to be a python package.